tutorials/introductory/patchwork_axes.py

```diff
@@ -0,0 +1,195 @@
+# will be twice as wide as the Axes on the right and the Axes on the
+# right will be twice as tall as Axes on the left.  But, as they say
+# a picture is worth a thousand words:
+
+fig = plt.figure(constrained_layout=True)
+axd = fig.build_grid(layout)
+label_axes(axd)
+
+###############################################################################
+# The returned value from `.Figure.build_grid` is a dictionary mapping the
+# label you set to the axes
+
+print(axd)
+
+###############################################################################
+# If instead we wanted a 2x2 grid on the left and the same span on the right
+axd = plt.figure(constrained_layout=True).build_grid(
+    """
+ABC
+DEC
+"""
+)
+label_axes(axd)
+
+###############################################################################
+# If we do not want to fill in all the spaces with axes you can leave
+# it blank
+
+label_axes(
+    plt.figure(constrained_layout=True).build_grid(
+        """
+ aa
+cBB
+cBB
+ dd
+"""
+    )
+)
+
+
+###############################################################################
+# If you would rather use another letter (rather than space  ``' '``) to mark
+# the empty space use *empty_sentinel* to specify the character to use.
+
+label_axes(
+    plt.figure(constrained_layout=True).build_grid(
+        """
+aX
+Xb
+""",
+        empty_sentinel="X",
+    )
+)
+
+
+###############################################################################
+# This can be particularly useful if your editor strips trailing white space.
+#
+# Because these are there are string inputs any Unicode glyph is valid!
+
+label_axes(
+    plt.figure(constrained_layout=True).build_grid(
+        """
+αб
+ℝ☢
+"""
+    )
+)
+
+###############################################################################
+# This feature is built on top of `.gridspec` and you can pass the
+# same keyword arguments through (the same with `.Figure.subplots` as
+# well) see `.gridspec.GridSpec` for the full set of keywords
+
+label_axes(
+    plt.figure(constrained_layout=True).build_grid(
+        """
+xax
+bAc
+xdx
+""",
+        gridspec_kw={"height_ratios": [1, 3, 1], "width_ratios": [1, 3, 1]},
+        empty_sentinel="x",
+    )
+)
+
+###############################################################################
+# You can also pass through arguments used to create the subplots
+# (again, the same as `.Figure.subplots`).
+
+label_axes(
+    plt.figure(constrained_layout=True).build_grid(
+        "AB", subplot_kw={"projection": "polar"}
+    )
+)
+
+
+###############################################################################
+# List input
+# ==========
+#
+# While the mulit-line string specification covers many use cases, we may want
+# to use more than a single letter to label our axes.
+
+axd = label_axes(
+    plt.figure(constrained_layout=True).build_grid(
+        [["main", "main", "callout"], ["main", "main", "BLANK"]],
```

Comment:
Breaking the list would make the correlation clearer.
```suggestion
        [["main", "main", "callout"],
         ["main", "main", "BLANK"]],
```